python/tox.ini

```diff
@@ -65,7 +68,7 @@ commands =
 deps =
     mypy
 commands =
-    mypy --no-implicit-optional src
+    mypy --install-types --non-interactive --no-implicit-optional --ignore-missing-imports src
```

Comment:
Wondering if we need this one `--ignore-missing-imports` for all cases? It might be better that we just ignore the ones that we know it is OK to ignore.